### PR TITLE
add notebook check for set_content (#124)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Solve Issue #124

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]
👍

File: taipy/gui/page.py [hunk 1]
```diff
@@ -47,6 +47,8 @@ def set_content(self, content: str) -> None:
 
         If _content_ is a path to a readable file, the file is read entirely as the text template.
         """
+        if not _is_in_notebook():
+            raise RuntimeError("set_content function should only be used inside iPython Notebook")
```
Comment (FabienLelaquais) [inline taipy/gui/page.py]: That would be 'must' instead of 'should'

**Review by FabienLelaquais — APPROVED** [review]
A tiny comment.
Great!

And please please update the doc!

File: taipy/gui/page.py (hunk 1 above)
Comment (dinhlongviolin1) [reply taipy/gui/page.py]: Thanks!

**Pull request merged**